container-search-gui/.gitignore

```diff
@@ -1,2 +1,3 @@
 /pom.xml.build
 /target
+/src/main/resources/gui/_includes/search-api-reference.html
```

Comment:
Could you generate to target/generated-sources/ instead? That is the standard location used inn maven projects.

Reply:
Removing this ignore, when search-api-reference.html now gets placed in /target